### PR TITLE
Route parameters should be plural for consistency

### PR DESCRIPTION
**Pull request by richiebres**
Singular route parameters will still work for the foreseeable future,
since they have already been released.

Both of these are valid:
* `/v1/apps/app_id/builds/`
* `/v1/apps/app_id/build/`

Ditto for crash reports:
* `/v1/apps/app_id/crash-reports/`
* `/v1/apps/app_id/crash-report/`

And deployment groups:
* `/v1/apps/app_id/deployment-groups/`
* `/v1/apps/app_id/deployment-group/`

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://17-bb-apidocs.surge.sh

File: builds/post-trigger.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ----
 {
   "base":     "\https://api.buddybuild.com/v1",
-  "endpoint": "/apps/:app_id:/build",
+  "endpoint": "/apps/:app_id:/builds",
```
Comment (bbewan) [inline builds/post-trigger.adoc]: Can you trigger multiple builds with one call to this endpoint? If not, this should be singular.

File: builds/get-latest_build.adoc [hunk 2]
```diff
@@ -5,7 +5,7 @@
 ----
 {
   "base":     "\https://api.buddybuild.com/v1",
-  "endpoint": "/apps/:app_id:/build/latest",
+  "endpoint": "/apps/:app_id:/builds/latest",
```
Comment (bbewan) [inline builds/get-latest_build.adoc]: Shouldn't `apps` and `builds` both be singular here? We show **the** latest build for **an** app, not multiple builds for multiple apps,?

File: crash_reports/get-logs.adoc [hunk 3]
```diff
@@ -5,7 +5,7 @@
 ----
 {
   "base":     "\https://api.buddybuild.com/v1",
-  "endpoint": "/apps/:app_id:/crash-report/:crash_report_id:/logs",
+  "endpoint": "/apps/:app_id:/crash-reports/:crash_report_id:/logs",
```
Comment (bbewan) [inline crash_reports/get-logs.adoc]: I think `crash-report` should be singular here. This endpoint fetches the logs from a specific crash report, right?

File: deployment_groups/delete-testers.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@
 ----
 {
   "base":     "\https://api.buddybuild.com/v1",
-  "endpoint": "/apps/:app_id:/deployment-group/:deployment_group_id:/testers",
+  "endpoint": "/apps/:app_id:/deployment-groups/:deployment_group_id:/testers",
```
Comment (bbewan) [inline deployment_groups/delete-testers.adoc]: I think `deployment-groups` should be singular here. We're deleting the testers from a specific deployment group, right?

File: deployment_groups/post-deploy.adoc [hunk 5]
```diff
@@ -5,7 +5,7 @@
 ----
 {
   "base":     "\https://api.buddybuild.com/v1",
-  "endpoint": "/:appID:/deployment-group/:deploymentTargetID:/deploy",
+  "endpoint": "/:appID:/deployment-groups/:deploymentTargetID:/deploy",
```
Comment (bbewan) [inline deployment_groups/post-deploy.adoc]: I think `deployment-groups` should be singular here, unless this endpoint can be used to deploy multiple deployment groups in one call.

File: deployment_groups/put-add_testers.adoc [hunk 6]
```diff
@@ -5,7 +5,7 @@
 ----
 {
   "base":     "\https://api.buddybuild.com/v1",
-  "endpoint": "/apps/:app_id:/deployment-group/:deployment_group_id:/testers",
+  "endpoint": "/apps/:app_id:/deployment-groups/:deployment_group_id:/testers",
```
Comment (bbewan) [inline deployment_groups/put-add_testers.adoc]: I think `deployment-groups` should be singular here.

**Comment by clintam** [comment]
Ah I see https://github.com/BuddyBuild/BB/pull/9418/files

Why not feedbacks too?

**Comment by clintam** [comment]
My take is that our notion of feedback is "countable" and hence we pluralize it with s. However I understand it sounds odd to english speaker, but think the REST consistency is more important.

**Comment by bbewan** [comment]
https://www.quora.com/What-is-the-plural-of-feedback-Can-you-ever-use-feedbacks

**Comment by clintam** [comment]
right @bbewan I can work a google too :) did you read that/see the EDIT?

I can count exactly the number of feedback*s* we have for an app!

**Comment by eskwayrd** [comment]
Fair point. `feedbacks` it is then. Any concern about `coverage`?

There are other consistency issues in our API. We violate the `use nouns, not verbs` best practice with `deploy` and `cancel`. I'm not sure whether those should be addressed too.

**Comment by bbdocs** [comment]
See the website here: http://17-bb-apidocs.surge.sh

**Pull request merged**